server/views/applications/pages/current-offences/current-offence-data.njk

```diff
@@ -0,0 +1,182 @@
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "../../../components/formFields/form-page-date-input/macro.njk" import formPageDateInput %}
+{% from "../../../components/formFields/form-page-text-area/macro.njk" import formPageTextArea %}
+
+{% extends "../layout.njk" %}
+
+{% set outstandingChargesFollowUp %}
+{{
+      formPageTextArea(
+        {
+          fieldName: 'outstandingChargesDetail',
+          label: {
+            text: page.questions.outstandingChargesDetail.question,
+            classes: "govuk-label--s"
+          }
+        },
+        fetchContext()
+      )
+    }}
+{% endset %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
```

Comment:
perhaps one that could be picked up as a snag, and our other 'data' pages probably don't have this either, but noticed this page doesn't have a `Back` or `back to Task list` link/button.

Reply:
yes, good idea. Looks like the other ones don't have them either - I'll add a ticket to the snag list